Makefile

```diff
@@ -34,7 +34,7 @@ pytest:
 .PHONY: black
 black:
 	${DOCKER_COMPOSE} \
-		run nornir black --check .
```

Comment:
why these changes?

Reply:
The current dockerfile is built in such a way that, it is mapping the whole project dir to the container, which causes the problems for my setup when I run tests... Decided to change to include directories explicitly when black should run.  
I can change it back, but I'd strongly advocate to change volume mounting in docker-compose then.

Reply:
if that's the case let's discuss it in a separate PR to avoid noise